MergePythonSDK/accounting/requirements.txt

```diff
@@ -0,0 +1,3 @@
+python_dateutil >= 2.5.3
```

Comment:
Please delete the `requirements.txt` files inside each category as they are cruft